packages/dds-tokens/build.js

```diff
@@ -28,13 +28,16 @@ StyleDictionary.registerFormat({
   format: customSCSSFormat,
 });
 
+const sourcePathBase = 'dds/tokens';
+const buildPathBase = 'build';
```

Comment:
Synes det er litt forvirrende at vi ender opp med både en `dist`- og `build`-mappe. Kanskje de bygde tokenene kan ligge i `generated-tokens` eller noe sånt 🤔 
```suggestion
const buildPathBase = 'generated-tokens';
```

Reply:
Ja, ser den. Bør nok bruke "generate" som uttrykk generelt for å skille fra build. Fikser.